e2e/validation_test.go

```diff
@@ -7,18 +7,22 @@ import (
 	"testing"
 
 	"github.com/ramendr/ramen/e2e/test"
-	"github.com/ramendr/ramen/e2e/util"
+	"github.com/ramendr/ramen/e2e/validate"
 )
 
 func TestValidation(dt *testing.T) {
 	t := test.WithLog(dt, Ctx.log)
 	t.Parallel()
 
+	if err := validate.TestConfig(Ctx.env, Ctx.config, Ctx.log); err != nil {
+		t.Fatalf("Failed to validate test environment with DR configurations: %s", err)
```

Comment:
Again, creating a good error in validateTestConfig means we can just use t.Fatal(err) here.